src/components/ec-smart-table/ec-smart-table.story.js

```diff
@@ -99,6 +100,12 @@ stories
       fetchEmptyList: {
         default: boolean('fetchEmptyList', false),
       },
+      defaultSortCycle: {
+        default: select('Sort Cycle by default', {
```

Comment:
Use name of the prop as a label just like the other props do.

Reply:
- [ ] Rename